auraed/src/lib.rs

```diff
@@ -268,6 +274,17 @@ impl AuraedRuntime {
 
         // Initialize the bundler
 
+        // Install eBPF probes
+        info!("Installing eBPF probes");
+
+        let bpf_loader = &mut BpfLoader::new()?;
+        let signals = bpf_loader.load_tracepoint::<Signal>(
```

Comment:
I'd prefer is we had a fn on `BpfLoader` named `load_signals_tracepoint`

Reply:
I was hoping to keep the `BpfLoader` agnostic of which actual programs are loaded and just supply it with a couple of methods like `load_tracepoint`, `load_kprobe`, `load_uprobe` etc and maintain a list of the actual programs we want to load elsewhere. I can add an extra function for now until we figure out where this list would live.

Reply:
I understand your reasoning and would agree if this were library code, but since this is internal I think the helper methods are the right approach. Thanks for making the change.